docs/faq.md

```diff
@@ -3,11 +3,48 @@ title: FAQs
 sidebar_position: 1000
 ---
 
+## General
+
+### How should I refer to Connect in a blog post or social media?
+
+**Connect** is the name of the project. If that's ambiguous in your context or
+search ranking/uniqueness is important, please use **Connect RPC**.
+
+### How do I get the OpenAPI spec for the Connect APIs?
+
+Connect RPCs are defined using Protobuf, which serves as the specification and
+documentation. OpenAPI is meant for REST/HTTP endpoints and doesn't apply to RPC
+systems like Connect.
+
 ## The Connect protocol
 
 ### Where can I find Connect implementations?
 
-All implementations are listed at the main [ConnectRPC repository][main-repo].
+All implementations are listed at the main [Connect RPC repository][main-repo].
+
+### Does Connect implement REST?
+
+Connect is an RPC protocol. It doesn't use HTTP verbs to model CRUD, and doesn't use
+resource-based URL routing, but a Connect service can meet the architectural constraints
+of a RESTful API. If you need CRUD verbs and resource-based URL routing, take a look at
+[Vanguard-Go](https://github.com/connectrpc/vanguard-go).
+
+### Is there a way to generate REST paths with Connect?
+
+The path is simply the fully qualified name of the service. There is no built-in option to
+modify this path, because it would break compatibility with gRPC and gRPC-Web.
+
+If you want more control over the path, one option would be to use
+[gRPC Transcoding](https://github.com/googleapis/googleapis/blob/738ff24cb9c00be062dc200c10426df7b13d1e65/google/api/http.proto#L44),
+which allows you to specify REST paths. You could use
+[vanguard-go](https://github.com/connectrpc/vanguard-go?tab=readme-ov-file#why-vanguard) or the
+[gRPC-Gateway](https://github.com/grpc-ecosystem/grpc-gateway#readme) to serve it.
```

Comment:
These two overlap. I think the first one is the better answer, and that we can remove the second.